docs/SRS/SRS.tex

```diff
@@ -127,7 +127,8 @@
 
 \begin{document}
 
-\title{Software Requirements Specification for \progname: subtitle describing software}
+\title{Software Requirements Specification for \progname: Progressive Web Application for Independent
```

Comment:
I was going to just bundle this in my other PR but we can do it separately. I think of a subtitle of something like "A Marketplace for Automotive Repair Industry".

Also I recommend adding a line break here to ensure this is on a separate line. Right now it is wrapping it because "Progressive" is a long word, but just case we change it.
```suggestion
\title{Software Requirements Specification for \progname:\\Progressive Web Application for Independent
```

Reply:
Actually it's fine the way it is.